model/src/main/java/org/cloudfoundry/identity/uaa/zone/Links.java

```diff
@@ -98,25 +98,35 @@ public Logout setWhitelist(List<String> whitelist) {
     }
 
     public static class SelfService {
-        private boolean selfServiceResetPasswordEnabled = true;
-        private boolean selfServiceCreateAccountEnabled = true;
+        private boolean selfServiceLinksEnabled = true;
+        private String selfServiceResetPasswordEnabled = null;
```

Comment:
why we converted type of selfServiceResetPasswordEnabled and  selfServiceCreateAccountEnabled to String?

Reply:
To accommodate all the backward compatibility use cases and selfservice split flags for new zones creation cases, to have the three different value for selfServiceResetPasswordEnabled and selfServiceCreateAccountEnabled as null, "true", "false".

Reply:
You can use wrapper type Boolean for all three values null, true and false.

Reply:
Done. Thanks!